docs/source/tutorial/project_setup.rst

```diff
@@ -0,0 +1,42 @@
+==============
+Project Setup
+==============
+
+If you haven't done so already make sure to complete the installation before moving on.
+
+Create Your Ingest Project
+==========================
+
+Create a new ingest project named ``My_Study`` by using the ingest library CLI::
+
+$ kidsfirst new My_Study
```

Comment:
I know we don't have this yet, but if possible we really should try to get this in before we roll out the MVP extract. I have an issue for it here #70

Reply:
We kinda have it in the tests directory already.

anyway, not

```
└
└
```